app/src/main/java/com/babylon/wallet/android/data/dapp/IncomingRequestRepository.kt

```diff
@@ -68,25 +71,71 @@ class IncomingRequestRepositoryImpl @Inject constructor() : IncomingRequestRepos
         }
     }
 
+    /**
+     * There are two path of execution when using this method:
+     * - there is high priority screen in the queue, so the incoming request is added below it,
+     * taking priority over requests currently in the queue
+     * - there is no high priority screen: request is added at the top of queue and if there other request currently handled,
+     * we send a dismiss event for it so that UI can react and dismiss handling, without removing it from the queue.
```

Comment:
"_there is no high priority screen: request is added at the top of queue and if there other request currently handled, ..._"

here you mean that if wallet receives another high priority request while another non high priority is being handled then we dismiss it and so on... right?

Reply:
Actually what I am thinking is that the term `dismiss` is not the right one to describe this particular business logic. "Dismiss" is like reject. We reject this request and we will never see it again. 

But what if we use `postpone` which describes that we delay the handling of this request? 
so it becomes `override suspend fun requestPostponed` and any other "dismiss" rename to "postpone"?

I would like to hear the opinion of others. Also any other word is welcome!

Reply:
oh now I think `defer` is even better. 
`override suspend fun requestDeferred`

Reply:
yes, whenever this method is used, request added with it is supposed to go on screen while current one is deferred. This does not apply if we are in flows like creating accounts or other flows marked as high priority.

Reply:
> This does not apply if we are in flows like creating accounts or other flows marked as high priority.

wait I think I lost it here, you mean we really dismiss the request in this case?

Reply:
nice thank you